netlify-webhook-deploy.php

```diff
@@ -132,15 +178,15 @@ public function plugin_settings_subpage_content() {?>
     public function run_the_mighty_javascript() {
         // TODO: split up javascript to allow to be dynamically imported as needed
         // $screen = get_current_screen();
-        // if ( $screen && $screen->parent_base != 'deploy_webhook_fields' && $screen->parent_base != 'deploy_webhook_fields_sub' ) {
+        // if ( $screen && $screen->parent_base != 'developer_webhook_fields' && $screen->parent_base != 'deploy_webhook_fields_sub' ) {
         //     return;
         // }
         ?>
         <script type="text/javascript" >
         console.log('run_the_mighty_javascript');
         jQuery(document).ready(function($) {
             var _this = this;
-            $( ".webhook-deploy_page_deploy_webhook_fields_sub td > input" ).css( "width", "100%");
+            $( ".webhook-deploy_page_developer_webhook_fields_sub td > input" ).css( "width", "100%");
```

Comment:
On the "Developer Settings" tab, the input fields no longer has 100% width. 

To fix this, this line must be changed back to:
```suggestion
            $( ".webhook-deploy_page_developer_webhook_fields td > input" ).css( "width", "100%");
```

Reply:
Would be nice to set this not to 100%, but not to teeny tiny. Something for the future!

Reply:
Yes, probably something for the future :-)